decentraland/unity-explorer
# fix: improves emotes visibility rules

## What does this PR change?

Fixes #1847 
Fixes #1824 

- Carnaval Muse: added a rule that disables all renderers on cases that the emote has only one clip. We assume is a v1 emote, which does not include any props, so it doesn't need to render anything.
- Feelin' Myself: reconverted asset bundle.
- Planting Seeds: the emote is not following [name conventions](https://docs.decentraland.org/creator/emotes/props-and-sounds/#naming-conventions) for the `_prop`. Creator needs to fix the problem.
- Triangle Tryout: considered as a v1 emote rule, so all renderers were disabled.
- Siesta en la Fiesta: the emote is not following [name conventions](https://docs.decentraland.org/creator/emotes/props-and-sounds/#naming-conventions) for the `_avatar`. Creator needs to fix the problem.

Also a new program arg has been added `--self-force-emotes` so we can force our avatar to have custom emotes.

## How to test the changes?



1. Launch the explorer with this command: `--self-force-emotes urn:decentraland:matic:collections-v2:0xa80aea22d0fe9d34ca72ce304ef427bbefee1f11:2,urn:decentraland:matic:collections-v2:0x5f6ca7e5c8055c4cb8736d065f619cdd0cf28752:4,urn:decentraland:matic:collections-v2:0xca53b9436be1d663e050eb9ce523decbc656365c:1,urn:decentraland:matic:collections-v2:0xe79cadd6fb15a626a48d2bbb60d4450eb196c84d:9,urn:decentraland:matic:collections-v2:0xfcc2c46c83a9faa5c639e81d0ad19e27b5517e57:0`
2. Open the emote wheel and execute them
3. Check that either base or on-chain emotes keep working as expected, specially v2 (those with props or particles)

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
